test/test_quantized.py

```diff
@@ -531,6 +626,52 @@ def test_adaptive_avg_pool2d(self, X, output_size_h, output_size_w):
                              message=error_message.format(name + '.zero_point', scale,
                                                           qX_hat.q_zero_point()))
 
+    """Tests adaptive average pool operation on NHWC quantized tensors."""
+    @given(X=hu.tensor(shapes=hu.array_shapes(min_dims=4, max_dims=4,
+                                              min_side=1, max_side=10),
+                       qparams=hu.qparams(dtypes=torch.qint8)),
```

Comment:
This is only being tested on `torch.qint8` because of this line. Can you omit the `dtypes` kwarg?